### PR TITLE
Bug 1707877: Use internal url only for internal load balancer

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

File: pkg/operator/certrotationcontroller/externalloadbalancer.go [hunk 1]
```diff
@@ -18,7 +18,6 @@ func (c *CertRotationController) syncExternalLoadBalancerHostnames() error {
 	hostname := infrastructureConfig.Status.APIServerURL
 	hostname = strings.Replace(hostname, "https://", "", 1)
 	hostname = hostname[0:strings.LastIndex(hostname, ":")]
-	hostname = strings.Replace(hostname, "api-int.", "api.", 1)
```
Comment (abhinavdahiya) [inline pkg/operator/certrotationcontroller/externalloadbalancer.go]: I think we need to keep this until we change the installer to set `infrastructureConfig.Status.APIServerURL` to `api`

**Comment by deads2k** [comment]
relies on the one after openshift/installer#1718

/lgtm

**Comment by abhinavdahiya** [comment]
> relies on the one after [openshift/installer#1718](https://github.com/openshift/installer/pull/1718)
> 
> /lgtm

https://github.com/openshift/installer/pull/1727 is the installer PR that moves to adding the public URL.

**Comment by wking** [comment]
/retest

openshift/installer#1718 has landed.

**Comment by deads2k** [comment]
Guys, it needs https://github.com/openshift/installer/pull/1727

**Comment by deads2k** [comment]
Until https://github.com/openshift/installer/pull/1727 merged the data in the cluster is wrong

**Comment by wking** [comment]
[e2e-aws][1]:

```
Installing from release registry.svc.ci.openshift.org/ci-op-tirqdfp4/release@sha256:4182c3386a92e18a53825b9da384128e729435214495a5b26a6b1657779f35d5
level=fatal msg="failed to fetch Terraform Variables: failed to fetch dependency of \"Terraform Variables\": failed to fetch dependency of \"Bootstrap Ignition Config\": failed to fetch dependency of \"Common Manifests\": failed to generate asset \"DNS Config\": getting public zone for \"origin-ci-int-aws.dev.rhcloud.com\": listing hosted zones: Throttling: Rate exceeded\n\tstatus code: 400, request id: 33e3ae82-71d6-11e9-adb1-e5c7a3ec66ff"
```

/test e2e-aws

[1]: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/464/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-aws/2211

**Comment by wking** [comment]
Dunno why my `/test` didn't stick.  Maybe this will:

/retest

**Comment by deads2k** [comment]
> Dunno why my `/test` didn't stick. Maybe this will:
> 
> /retest

github has had trouble with time today.  People have been writing to us from the future and the bots go a little nuts

**Comment by wking** [comment]
```
2019/05/08 23:09:05 Pod e2e-aws succeeded after 1h0m50s
```
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/464/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-aws/2213/build-log.txt

So any time now, Prow will notice and land this :)

**Comment by wking** [comment]
Queued for retest <- noooo! https://prow.svc.ci.openshift.org/pr?query=is%3Apr+repo%3Aopenshift%2Fcluster-kube-apiserver-operator+author%3Adamemi+head%3Ause-internal-url

**Comment by wking** [comment]
Wait, #465 looks related?  Did we need that too for the beta blocker?

**Comment by wking** [comment]
No, #465 merged ages ago.  What are you doing, Tide?  Can we just green-button this?

**Comment by deads2k** [comment]
> No, #465 merged ages ago. What are you doing, Tide? Can we just green-button this?

Is there some driver forcing that?  I'd rather not.  At this stage, with this risk profile, dependent on a recently merged change in another repo, I'd like to be sure of the green.  There's nothing else in queue for this repo, I'm inclined to wait. until I see mostly green status.

**Pull request merged**